checkbox-ng/checkbox_ng/launcher/test_subcommands.py

```diff
@@ -113,3 +287,43 @@ def test_invoke_returns_1_on_many_diff_outcomes(self):
         mock_results = {"fail": 6, "crash": 7, "pass": 8}
         self.ctx.sa.get_summary = Mock(return_value=mock_results)
         self.assertEqual(self.launcher.invoked(self.ctx), 1)
+
+
+class TestUtilsFunctions(TestCase):
+    @patch("checkbox_ng.launcher.subcommands.Colorizer", new=MagicMock())
+    @patch("builtins.print")
+    @patch("builtins.input")
+    def test_request_comment(self, input_mock, print_mock):
+        input_mock.side_effect = ["", "failure"]
+
+        comment = request_comment("Job Name")
+
+        self.assertEqual(comment, "failure")
+
+    def test__generate_resume_candidate_description_default_time(self):
+        candidate_mock = MagicMock()
+        candidate_mock.metadata.app_blob = '{ "testplan_id" : "123" }'
+        candidate_mock.metadata.title = "Title"
+        candidate_mock.metadata.last_job_start_time = None
+        candidate_mock.metadata.running_job_name = "Test"
+
+        description = _generate_resume_candidate_description(candidate_mock)
+
+        self.assertIn("Unknown", description)
+        self.assertIn("123", description)
+        self.assertIn("Title", description)
+        self.assertIn("Test", description)
+
+    def test__generate_resume_candidate_description(self):
+        candidate_mock = MagicMock()
+        candidate_mock.metadata.app_blob = '{ "testplan_id" : "123" }'
+        candidate_mock.metadata.title = "Title"
+        candidate_mock.metadata.last_job_start_time = 1
```

Comment:
This is a bit weird. I would have a concrete date and parse it to a timestamp using `datetime`

```
        # Create a datetime object for a specific date (e.g., January 1, 2023)
        date = datetime.datetime(2023, 1, 1)
        candidate_mock.metadata.last_job_start_time = date.timestamp()
...
        self.assertIn("2023", description)
```

Reply:
The code you proposed would work only in one timezone

```
Python 3.11.6 (main, Oct  8 2023, 05:06:43) [GCC 13.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import datetime
>>> dt1 = datetime.datetime(2023,1,1)
>>> dt1.timestamp()
1672527600.0
>>> dt2 = datetime.datetime(2023,1,1, tzinfo=datetime.UTC)
>>> dt2.timestamp()
1672531200.0
```

Reply:
Ohh, that's true. I didn't saw that we are using `dt = datetime.utcfromtimestamp(last_job_timestamp)` to parse the timestamp in `subcomands.py` instead of `fromtimestamp()` It makes sense they yield different results.